app/models/handoff/CompanyNameHandOffInformation.scala

```diff
@@ -21,6 +21,8 @@ import java.util.TimeZone
 import org.apache.commons.lang3.time.FastDateFormat
 import org.joda.time.DateTime
 import play.api.libs.json._
+import play.api.libs.json.JodaWrites._
+import play.api.libs.json.JodaReads._
```

Comment:
intellij shows them unused ?

Reply:
Yeah - this is strange. It shows them as unused but without them there it fails to compile